scripts/ci/node-template-release/src/main.rs

```diff
@@ -8,23 +6,28 @@ use std::{
 	process::Command,
 };
 
-use glob;
-
+use clap::Parser;
```

Comment:
How does this work?  
What is this `node-template-release` being used for? Is it for the developer hub?

Reply:
It rips off the `node-template` from the Substrate repo, creates a workspace Cargo.toml and zips the resulting files.